test/index.test.js

```diff
@@ -456,6 +456,11 @@ describe("PinkyPromise fake time tests", () => {
 		).resolves.not.toThrow();
 		expect(callback).toHaveBeenCalledTimes(2);
 
+		const cancelCallback1 = jest.fn();
```

Comment:
Can we move this into a new `it()` block?

Reply:
Moved to it's own test